hack/remove-calico-policy/README.md

```diff
@@ -1,4 +1,41 @@
-# Disabling and Removing Calico Policy
+# Disabling Calico Policy
+
+This guide explains two methods for disabling Calico policy on a running cluster. These steps are intended as a
+last resort should it appear that Calico policy is not functioning properly on a production system.
+
+- [Overriding Calico Policy] describes how to add high-priority Calico System Network Policy which
+  overrides the configured Network Policy.
+- [Disabling and Removing Calico Policy] describes how to fully disable Calico policy (removing all 
+  Calico-programmed policy from each node.
+
+## Overriding Calico Policy
+
+This section describes how to override the Calico-enabled Network Policy using higher priority system-wide
+network policy.  This system-wide policy is implemented using a `SystemNetworkPolicy` Third Party Resource.
+
+Instances of these resources created in the `kube-system` namespace will be interpreted by Calico and applied
+as system-wide Calico Policy.
+
+In the following we describe how to create a `SystemNetworkPolicy` resource using `kubectl` to allow all
+ingress and egress traffic.
+
+### Requirements / Assumptions
+
+- Calico version v2.3 or higher
+- Kubernetes v1.6 or higher
+- These steps assume Calico is running in policy-only mode (without Calico networking)
+
+### Instructions
+
+> **Note:** The following steps assume you have permissions to create resources in the `kube-system` namespace.
+
+#### 1. Create a `SystemNetworkPolicy` resource
+
+```
+kubectl create --namespace=kube-system --filename=./system-network-policy-override.yaml
```

Comment:
Sure - I was trying to be consistent with the existing instructions.

So you want the other instructions updated too?